packages/legacy/core/App/screens/ProofRequest.tsx

```diff
@@ -458,7 +460,7 @@ const ProofRequest: React.FC<ProofRequestProps> = ({ navigation, route }) => {
                 </View>
               )}
             </View>
-            {!hasAvailableCredentials && (
+            {!hasAvailableCredentials && hasSomeCred && (
```

Comment:
Can we rename `hasSomeCred` to somthing more clear like `hasMatchingCredDef` or `hasCredWithDefId`?